ansible/roles/detect-ssd/tasks/main.yaml

```diff
@@ -0,0 +1,14 @@
+---
+  - debug:
+      msg: "Checking if the disk {{ satellite_physical_disk }} is SSD"
+  - name: Retrieve the contents of the disk
+    shell:
+      cat "/sys/block/{{ satellite_physical_disk }}/queue/rotational
```

Comment:
Given a default, this would expand into "/sys/block//dev/sda/queue/rotational" and that does not seems right

Reply:
@jhutar I am a little bit confused here on which approach to take:
- We can have a default disk set in the configuration, but that may cause mis-representation of facts if the user does not take care while running the tune utility.
- We can have this variable set to a blank value and we can check in playbook, if the variable is blank, then don't run this detection, otherwise run this detection mechanism